Jenkinsfile.deploy

```diff
@@ -70,9 +70,30 @@ node('put-shared') { ansiColor('xterm') {
 					include "deploy";
 					arch_tagged_manifests(env.BASHBREW_ARCH)
 					| deploy_objects[]
-				' builds.json | tee deploy.json
+				' builds.json > deploy.json
 			'''
 		}
+		stage('Filter') {
+			// using the previous successful deploy.json, filter the current deploy.json with items already pushed lastime
+			sh'''
+				if \\
+					! wget --timeout=5 -qO past-deploy.json "$JOB_URL/lastSuccessfulBuild/artifact/deploy.json" \\
+					|| ! jq 'empty' past-deploy.json \\
+				; then
+					touch past-deploy.json
```

Comment:
With the second half of the conditional checking for valid JSON, this should probably do `truncate` with a zero size instead, right? (or we remove that check and let the jobs fail until we take explicit action, which is maybe safer since it'd be something unexpected and failing is easier to notice than slower)

Reply:
Oh, yeah. let me adjust it. 🤦

Reply:
🤔 We only need the `if ! wget` for the first time it is run.  After that, does it make sense to fail if there isn't one? So, just as committed in the PR or perhaps without the `jq` test the first time and once they all run, drop the `if` and just let `wget` fail?
```bash
				if ! wget --timeout=5 -qO past-deploy.json "$JOB_URL/lastSuccessfulBuild/artifact/deploy.json"; then
					touch past-deploy.json
				fi
```

Reply:
Hmm yeah, we could do that, although if we ever need to bootstrap again it's a little bit of a hassle.

Reply:
We could even start with the raw failing `wget` version, perhaps leaving the `touch` version commented so we have an easier "replay" target and use replay to bootstrap all of them the first time?

Reply:
(then we also have an obvious place to leave a comment right where we might investigate in the future, describing how if anything goes wrong here and we need to re-bootstrap, here's how to do it)

Reply:
I have updated it to fail if the `wget` fails with a note and `touch` command for bootstrapping.